x-pack/plugins/integration_assistant/server/templates/manifest/aws_cloudwatch_manifest.yml.njk

```diff
@@ -1,7 +1,9 @@
 - input: aws-cloudwatch
   template_path: aws-cloudwatch.yml.hbs
-  title: {{ data_stream_title }}
-  description: {{ data_stream_description }}
+  title: |
```

Comment:
This change is needed to prevent free text parameters from breaking the yml format when they contain some special character such as `:` or `-`. This same fix has been applied to all templates

(The `data_stream_name` parameter is not free text, we don't need to do this)